packages/dom/src/phrasing-content.js

```diff
@@ -20,7 +20,7 @@ const textContentSchema = {
 	s: {},
 	del: {},
 	ins: {},
-	a: { attributes: [ 'href', 'target', 'rel' ] },
+	a: { attributes: [ 'href', 'target', 'rel', 'id' ] },
```

Comment:
If we allow ids for links, we should allow them for all tags, but only the content has a link to it.

Reply:
@ellatrix One question about this: why do you think it is important to allow 'id' for all elements? The scope of this issue is to fix the broken footnotes (and perhaps endnotes) when pasting from Word, and only anchor elements are involved.

Reply:
Because generally, any element could be linked to, for example a heading. I agree, let's add this to one element at a time though.